packages/meta-harness/lib/evaluator.py

```
@@ -658,24 +687,160 @@ def _verify_registered_overlay_integrity(manifest: dict, overlay_dir: Path) -> N
+                item["value"],
+            )
+        rendered = yaml.safe_dump(
+            local_config,
+            allow_unicode=True,
+            default_flow_style=False,
+            sort_keys=True,
+        )
+        _atomic_write_worktree_file(local_path, rendered)
+
+    # 適用済み patch の内容を worktree 内の固定パスへ記録する。scenario の command_exit
+    # oracle は隔離実行され、worktree の外（meta-harness store 上の overlay/config-patch.json
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)